### DISCUSSION TITLE
Make UI more consistent

### DISCUSSION THREAD

**Discussion by OneNiko727** [ppy/osu#29092] [Ideas]
At the current state of osu! lazer, the UI is quite messy. Some components look like osu! lazer from 2017 and others look like the current modern version of the UI. There's no need to overhaul things like the main menu yet, but small things like the now playing ui and main menu options tab sure do. Also the welcome menu needs a bit of recoloring, colors are quite bright and not consistent with the newer color scheme across new UI components.

**Comment by peppy** [comment]
the welcome screen already uses latest designs and colours, as does settings. you're going to have to explain your qualms in finer detail

**Reply by OneNiko727** [reply]
My mistake, I might've gotten a few things incorrect.
The welcome screen is just fine.
But, the options tab looks off. This is the options I was talking about. It just isn't up to date with the rest of the current icons.
![image](https://github.com/user-attachments/assets/6a2f4185-b330-44ac-a218-eabbc7fe3ac8)

**Reply by Stoppedpuma** [reply]
The footer has already been remade in tests and will eventually make it into the game.

![image](https://github.com/user-attachments/assets/edc940f7-d4e1-4e13-b8a5-caddb4299d10)

> It just isn't up to date with the rest of the current icons.

Do you expect the icons to match the icons found at the top bar as well as main menu screen? E.g replacing the edit beatmap icon with this icon?

![image](https://github.com/user-attachments/assets/4ceddef6-0318-4cb5-8d82-7c4b320fc554)

**Reply by OneNiko727** [reply]
holy I cant wait to see that UI in the production stage of the game
